src/CBLTLSIdentity_CAPI.cc

```diff
@@ -0,0 +1,201 @@
+}
+
+FLSliceResult CBLCert_Data(CBLCert* cert, bool pemEncoded) noexcept {
+    try {
+        return (FLSliceResult)cert->data(pemEncoded);
+    } catchAndBridge(nullptr);
+}
+
+FLSliceResult CBLCert_SubjectName(CBLCert* cert) noexcept {
+    try {
+        return (FLSliceResult)cert->subjectName();
+    } catchAndBridge(nullptr);
+}
+
+FLSliceResult CBLCert_SubjectNameComponent(CBLCert* cert, FLString attributeKey) noexcept {
+    try {
+        return (FLSliceResult)cert->subjectNameComponent(attributeKey);
+    } catchAndBridge(nullptr);
+}
+
+void CBLCert_getValidTimespan(CBLCert* cert,
+                              CBLTimestamp* outCreated,
+                              CBLTimestamp* outExpires) noexcept {
+    try {
+        cert->getValidTimespan(outCreated, outExpires);
+    } catchAndWarnNoReturn();
+}
+
+CBLKeyPair* CBLCert_PublicKey(CBLCert* cert) noexcept {
+    try {
+        return retain(cert->publicKey());
+    } catchAndBridge(nullptr);
+}
+
+// CBLTLSIdentity
+
+CBLTLSIdentity* CBLTLSIdentity_SelfSignedCertIdentity(bool server,
+                                                      CBLKeyPair* keypair,
+                                                      FLDict attributes,
+                                                      CBLTimestamp expiration,
+                                                      CBLError* outError) noexcept {
+    try {
+        return retain(CBLTLSIdentity::SelfSignedCertIdentity(server,
+                                                             keypair,
+                                                             attributes,
+                                                             expiration));
+    } catchAndBridge(outError);
+}
+
+CBLTLSIdentity* CBLTLSIdentity_IdentityWithKeyPairAndCerts(CBLKeyPair* keypair,
+                                                           CBLCert* cert,
+                                                           CBLError*  outError) noexcept {
+    try {
+        return retain(CBLTLSIdentity::IdentityWithKeyPairAndCerts(keypair, cert));
+    } catchAndBridge(outError);
+}
+
+CBLCert* CBLTLSIdentity_Certificates(CBLTLSIdentity* tlsID) noexcept {
+    return tlsID->certificates();
+}
+
+CBLTimestamp CBLTLSIdentity_Expiration(CBLTLSIdentity* tlsID) noexcept {
+    return tlsID->expiration();
+}
+
+#if !defined(__linux__) && !defined(__ANDROID__)
+
+CBLTLSIdentity* CBLTLSIdentity_SelfSignedCertIdentityWithLabel(bool server,
+                                                                FLString persistentLabel,
+                                                                FLDict attributes,
+                                                                CBLTimestamp expiration,
+                                                                CBLError* outError) noexcept {
+    try {
+        // TBD
+        return new CBLTLSIdentity;
+    } catchAndBridge(outError);
+}
+
+bool CBLTLSIdentity_deleteIdentityWithLabel(FLString persistentLabel,
```

Comment:
The implementation still has the old name. 😉

Reply:
yes, will fix it when actually implementing it. Thanks for pointing it out.